api/Clipboard.json

```
@@ -395,7 +395,8 @@
               "version_added": "47"
             },
             "safari": {
-              "version_added": "13.1"
+              "version_added": "13.1",
+              "notes": "Must be called within user gesture event handlers like pointerdown or pointerup"
```

A small bit of copy editing here ("such as" since I think these are included, not just similar to; code format)

```suggestion
              "notes": "Must be called within user gesture event handlers such as <code>pointerdown</code> or <code>pointerup</code>."
```